variables.tf

```diff
@@ -143,6 +143,12 @@ variable "enable_host_encryption" {
   default     = false
 }
 
+variable "enable_load_balancer_profile" {
+  type        = bool
+  description = "(Optional) Enable a load_balancer_profile block. This can only be used when load_balancer_sku is set to `standard`."
+  default     = false
```

Comment:
I'd like to restrict this variable as `nullable = false` since I don't think anyone should set this variable to `null`.